src/box2dx/Box2D.NetStandard/Dynamics/Contacts/ContactSolver.cs

```diff
@@ -780,7 +780,10 @@ internal void Initialize(ContactPositionConstraint pc, Transform xfA, Transform
                         {
                             Vector2 pointA = Common.Math.Mul(xfA, pc.localPoint);
                             Vector2 pointB = Common.Math.Mul(xfB, pc.localPoints[0]);
-                            normal = Vector2.Normalize(pointB - pointA);
+                            if (pointA == pointB)
+                                normal = new Vector2(0, 1);
+                            else
+                                normal = Vector2.Normalize(pointB - pointA);
```

Comment:
Can you please do one line:
```
normal = pointA == pointB ? Vector2.UnitY : Vector2.Normalize(pointB - pointA);
```
`Vector2.UnitY` returns `(0, 1)`.

Reply:
sure!